### DISCUSSION TITLE
Is it ok for contributors to check, review and change packages?

### DISCUSSION THREAD

**Discussion by GorujoCY** [microsoft/winget-pkgs#15704] [Q&A] [answered]
as the title implies, Is it ok for contributors to check and review pull requests? even ask for changes to necessarily make them work?
Hopefully that's a yes, because that would speed up the process of Pull requests if we aren't relying on automation.

**Comment by denelon** [comment, marked as answer]
Absolutely!

The community would welcome supportive suggestions. I just recently added a [CONTRIBUTING.md](https://github.com/microsoft/winget-pkgs/blob/master/CONTRIBUTING.md), and have started working on [AUTHORING_MANIFESTS.md](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md). We still have plenty of documentation needs and best practices to build.

Be sure to keep our [CODE_OF_CONDUCT.md](https://github.com/microsoft/winget-pkgs/blob/master/CODE_OF_CONDUCT.md) in mind when interacting with others. We're all human (except for @msftbot).
